cmd/helm/testdata/testcharts/novals/templates/alpine-pod.yaml

```diff
@@ -6,12 +6,12 @@ metadata:
     # The "heritage" label is used to track which tool deployed a given chart.
     # It is useful for admins who want to see what releases a particular tool
     # is responsible for.
-    heritage: {{.Release.Service | quote }}
+    app.kubernetes.io/managed-by: {{.Release.Service | quote }}
     # The "release" convention makes it easy to tie a release to all of the
     # Kubernetes resources that were created as part of that release.
-    release: {{.Release.Name | quote }}
+    app.kubernetes.io/instance: {{.Release.Name | quote }}
```

Comment:
How come this one not `helm.sh/release`: {{.Release.Name | quote }}?

Reply:
A helm release provides a unique name for an instance of a chart. This allows me to do things like run multiple versions of the same chart where each has a unique name. That's the same purpose for the `app.kubernetes.io/instance` label. This is not a unique problem to helm.

This plays into the realm of compatibility with other tools, too.

Imagine dashboard wants to display all the different instances of applications no matter what deployed it. This is a common label to group and work on the things.

Does that help?

Reply:
Yea that makes sense!